WooCommerce/WooCommerceTests/Notifications/PushNotificationsManagerTests.swift

```diff
@@ -82,8 +87,21 @@ final class PushNotificationsManagerTests: XCTestCase {
     /// Verifies that `resetBadgeCount` sets the badgeNumber to zero
     ///
     func testResetBadgeCountEffectivelyDropsTheBadgeNumberToZero() {
+        // Arrange
+        // The default stores are required to update the application badge number.
+        let stores = DefaultStoresManager.testingInstance
+        ServiceLocator.setStores(stores)
```

Comment:
Same as above, I don't think we need the `ServiceLocator.setStores` since the `storesManager` is injected.

Reply:
similar to above, updated in https://github.com/woocommerce/woocommerce-ios/pull/2320/commits/7620bb0da4f901ebfd141939e5511a9ab657c39e